packages/operator/pkg/service/toolchain/service.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package toolchain
+
+import (
+	"github.com/odahu/odahu-flow/packages/operator/pkg/apis/training"
+	training2 "github.com/odahu/odahu-flow/packages/operator/pkg/repository/training"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/utils/filter"
+	"time"
+)
+
+type Service interface {
```

Comment:
I'd prefer to declare all interfaces in Consumers of API, not in implementations. At least for new code. For old code we should refactor it later to fit best Golang practices